### PR TITLE
DB-2483: include only install addons in profile import

### PR DESCRIPTION
**Pull request by tsl143**

### CODE REVIEW

File: mozilla-release/browser/components/migration/FirefoxProfileMigrator.jsm [hunk 1]
```diff
@@ -580,16 +580,30 @@ FirefoxProfileMigrator.prototype._getResourcesInternal = async function(
...
   let getAddons = async function() {
     try {
-      const oldPath = OS.Path.join(sourceProfileDir.path, "addons.json");
+      const oldPath = OS.Path.join(sourceProfileDir.path, "extensions.json");
       const exists = await OS.File.exists(oldPath);
       if (exists) {
         let raw = await OS.File.read(oldPath, {encoding: "utf-8"});
         let data = JSON.parse(raw);
         if (data && data.addons && data.addons.length > 0) {
+          // Allow import of addons only form profile
```
[the remark this answers is not in the document]
Comment (tsl143) [reply mozilla-release/browser/components/migration/FirefoxProfileMigrator.jsm:589]: Its custom function from us ;)

File: mozilla-release/browser/components/migration/FirefoxProfileMigrator.jsm [hunk 2]
```diff
@@ -580,16 +580,30 @@ FirefoxProfileMigrator.prototype._getResourcesInternal = async function(
 
   let getAddons = async function() {
```
Comment (aleksei-cliqz) [inline mozilla-release/browser/components/migration/FirefoxProfileMigrator.jsm:581]:
```suggestion
  CLIQZ-SPECIAL
  let getAddons = async function() {
```

Comment (tsl143) [reply mozilla-release/browser/components/migration/FirefoxProfileMigrator.jsm:581]: Lets rename function instead as we do

File: mozilla-release/browser/components/migration/FirefoxProfileMigrator.jsm [hunk 3]
```diff
@@ -578,18 +578,33 @@ FirefoxProfileMigrator.prototype._getResourcesInternal = async function(
     };
   }.bind(this);
 
+  CLIQZ-SPECIAL
```
Comment (aleksei-cliqz) [inline mozilla-release/browser/components/migration/FirefoxProfileMigrator.jsm:581]:
```suggestion
  // CLIQZ-SPECIAL
```

**Pull request merged**